src/traces/indicator/attributes.js

```diff
@@ -0,0 +1,408 @@
+        min: 0,
+        max: 1,
+        dflt: 1,
+        editType: 'plot',
+        description: [
+            'Sets the thickness of the bar as a fraction of the total thickness of the gauge.'
+        ].join(' ')
+    },
+    editType: 'calc'
+};
+
+var stepsAttrs = templatedArray('threshold', extendDeep({}, gaugeBarAttrs, {
+    range: {
+        valType: 'info_array',
+        role: 'info',
+        items: [
+            {valType: 'number', editType: 'plot'},
+            {valType: 'number', editType: 'plot'}
+        ],
+        editType: 'plot',
+        // impliedEdits: {'autorange': false},
+        description: [
+            'Sets the range of this axis.',
+            'If the axis `type` is *log*, then you must take the log of your',
+            'desired range (e.g. to set the range from 1 to 100,',
+            'set the range from 0 to 2).',
+            'If the axis `type` is *date*, it should be date strings,',
+            'like date data, though Date objects and unix milliseconds',
+            'will be accepted and converted to strings.',
+            'If the axis `type` is *category*, it should be numbers,',
+            'using the scale where each category is assigned a serial',
+            'number from zero in the order it appears.'
+        ].join(' ')
+    }
+}));
+
+module.exports = {
+    mode: {
+        valType: 'flaglist',
+        editType: 'calc',
+        role: 'info',
+        flags: ['number', 'delta', 'gauge'],
+        dflt: 'number'
+    },
+    value: {
+        valType: 'number',
+        editType: 'calc',
+        role: 'info',
+        anim: true,
+        description: [
+            'Sets the number to be displayed.'
+        ].join(' ')
+    },
+    align: {
+        valType: 'enumerated',
+        values: ['left', 'center', 'right'],
+        role: 'info',
+        editType: 'plot',
+        description: [
+            'Sets the horizontal alignment of the `text` within the box.',
+            'Note that this attribute has no effect if an angular gauge is displayed:',
+            'in this case, it is always centered'
+        ].join(' ')
+    },
+    // position
+    domain: domainAttrs({name: 'indicator', trace: true, editType: 'calc'}),
+
+    vmin: {
+        valType: 'number',
+        editType: 'calc',
+        role: 'info',
+        dflt: 0,
+        description: [
+            'Sets the minimum value of the gauge.'
+        ].join(' ')
+    },
+    vmax: {
+        valType: 'number',
+        editType: 'calc',
+        role: 'info',
+        description: [
+            'Sets the maximum value of the gauge.'
+        ].join(' ')
+    },
+
+    title: {
+        text: {
+            valType: 'string',
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Sets the title of this indicator.'
+            ].join(' ')
+        },
+        align: {
+            valType: 'enumerated',
+            values: ['left', 'center', 'right'],
```

Comment:
I think `title.align` corresponds to what most users will find useful for `indicator` traces, but note that `align` somewhat conflicts with `textposition` in pie traces

https://github.com/plotly/plotly.js/blob/795220f815afcf79c73885715379b606afc1b4a1/src/traces/pie/attributes.js#L208-L222

and the `(x|y)ref` / `(x|y)anchor` attributes for `layout.title`:

https://github.com/plotly/plotly.js/blob/795220f815afcf79c73885715379b606afc1b4a1/src/plots/layout_attributes.js#L51-L130

and there's `side` in colorbars:

https://github.com/plotly/plotly.js/blob/795220f815afcf79c73885715379b606afc1b4a1/src/plots/layout_attributes.js#L51-L130

Reply:
I'm willing to either rename the attribute to `position` as in `pie` or change the API altogether. 

FYI, the `(title|number).align` attributes were added at the last minute in order to mimic DDK's "data cards" in which the text is left-aligned and position to the left of the domain.